proposals/34:upgrade-10/yarn.lock

```diff
@@ -5,6 +5,171 @@ __metadata:
   version: 8
   cacheKey: 10c0
 
+"@agoric/synthetic-chain@file:./agoric-synthetic-chain-0.0.1-alpha.tgz::locator=root-workspace-0b6124%40workspace%3A.":
+  version: 0.0.1-alpha
+  resolution: "@agoric/synthetic-chain@file:./agoric-synthetic-chain-0.0.1-alpha.tgz#./agoric-synthetic-chain-0.0.1-alpha.tgz::hash=1144d8&locator=root-workspace-0b6124%40workspace%3A."
+  dependencies:
+    tsx: "npm:^3.12.8"
+  bin:
+    synthetic-chain: ./cli.ts
+  checksum: 012b67bb80fc818bdb63a16f05007c7183a63d45d0a99a2ce1477079fe2249eae0961c6325b30cea111f4dd38487011f20a94a1518a8b8badef7c4583992d131
+  languageName: node
+  linkType: hard
+
+"@esbuild/android-arm64@npm:0.18.20":
```

Comment:
I see `esbuild` pop up in our builds from time to time. I wonder what depends on it.

Here's hoping for a moment to check.

Reply:
'tsx' does